runtime/CMakeLists.txt

```diff
@@ -120,19 +120,17 @@ install_helper(
   FILES ${CMAKE_CURRENT_SOURCE_DIR}/neovim.ico
   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime)
 
-globrecurse_wrapper(RUNTIME_PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR} *.awk *.sh *.bat)
-
-foreach(PROG ${RUNTIME_PROGRAMS})
-  get_filename_component(BASEDIR ${PROG} DIRECTORY)
-  install_helper(PROGRAMS ${PROG}
-                 DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime/${BASEDIR})
+glob_wrapper(RUNTIME_ROOT_FILES *.vim *.lua)
+foreach(F ${RUNTIME_ROOT_FILES})
+  install_helper(FILES ${F}
+                 DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime/)
 endforeach()
 
-globrecurse_wrapper(RUNTIME_FILES ${CMAKE_CURRENT_SOURCE_DIR}
-  *.vim *.lua *.scm *.dict *.py *.rb *.ps *.spl *.tutor *.tutor.json)
```

Comment:
I guess the idea was to avoid putting extraneous files that might exist in the build workspace into the install dir. But that should be the responsibility of `make clean` and friends.

Reply:
If this turns out to be a notable source of friction, I think we can use EXCLUDE instead, i e to exclude all files starting by `.` which covers inline swapfiles.